controllers/front/ProductController.php

```diff
@@ -87,6 +87,9 @@ public function init()
         if (!Validate::isLoadedObject($this->product)) {
             Tools::redirect('index.php?controller=404');
         } else {
+            if (!$this->product->hasCombinations()) {
+                $_GET['id_product_attribute'] = 0;
```

Comment:
Without it the URL looks really dirty.

With:      http://localhost/PrestaShop/home/8-2-test-no-combi.html
Without: http://localhost/PrestaShop/home/8-2-2-test-no-combi.html?id_product_attribute=2

Maybe there is a better way/place to do it? @maximebiloe @aleeks